app/scripts/views/settings/clients.js

```diff
@@ -45,28 +49,45 @@ define(function (require, exports, module) {
       }
 
       var devices = this._devices;
-      devices.on('add', this._onDeviceAdded.bind(this));
-      devices.on('remove', this._onDeviceRemoved.bind(this));
+      devices.on('add', this._onItemAdded.bind(this));
```

Comment:
When I first wrote the devices code, I didn't know about [listenTo](http://backbonejs.org/#Events-listenTo) which would allow you to do this:

```js
this.listenTo(devices, 'add', this._onItemAdded);
```

The context is set correctly, no need for bind, and when the view is destroyed the listeners are 
automatically removed.